contrib/docker/Dockerfile

```
@@ -28,10 +28,13 @@ RUN useradd --home-dir /srv/$ZCASHD_USER \
             --create-home \
             --uid $ZCASHD_UID\
             $ZCASHD_USER
```

```suggestion
            --uid "$ZCASHD_UID" \
            "$ZCASHD_USER"
```
Also `"$ZCASH_SIGNING_KEY_ID"` on line 16, `"$ZCASH_VERSION"` on line 22, and `"/srv/$ZCASHD_USER"` on line 26.